result/isislsp.yaml

```diff
@@ -0,0 +1,356 @@
+components:
+  schemas:
+    IsisLsps.State.Request:
+      description: >- 
+        The request to retrieve ISIS Link State Protocol Data Unit (LSP) information learned by this router.
+      type: object
+      properties:
+        isis_router_names:
+          description: >-
+            The names of ISIS routers for which learned information is requested.
+            An empty list will return results for all ISIS routers.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.IsisRouter/properties/name"
+  
+    IsisLsps.State:
+      description: >-
+        The result of ISIS LSP information that are retrieved.
+      type: object
+      properties:
+        isis_router_name:
+          description: >-
+            The name of the ISIS Router.
+          type: string
+        lsps:
+          description: >-
+            One or more LSPs that are learned by this ISIS router.
+          type: array
+          items:
+            $ref: '#/components/schemas/IsisLsp.State'
+
+    IsisLsp.State:
+      description: >-
+        ISIS LSP.
+      type: object
+      properties:
+        lsp_id:
+          description: >-
+            LSP ID of the LSP, in the format, e.g. '640000000001-00-00'.
```

Comment:
"LSP ID of the LSP, in the format" -> "LSP ID in the format"

Reply:
ok.